lib/Elastica/Index/Settings.php

```diff
@@ -103,6 +103,25 @@ public function get($setting = '')
     }
 
     /**
+     * Returns a setting interpreted as a bool.
```

Comment:
Nice addition. I hope that ES will also clean this up in the future and only have 1 option of returning bools.